_component_design/content-dividers.md

```diff
@@ -3,7 +3,8 @@ title: Content Dividers
 parent: null
 layout: component
 category: Components
-usage: Teaser Text
+usage: >
+  Content dividers are used within content areas to generate visual seperation. This seperation help to increase user understanding by segmenting information into distinct groups.
```

Comment:
Nit: `help` should be `helps`:
```
This separation helps to increase...
```

Reply:
Fixed in commit fe6a7cf and added to this PR.